tasks/sep/026-fp-holocene-upgrade-fix/input.json

```diff
@@ -0,0 +1,67 @@
+{
+  "chainId": 11155111,
+  "metadata": {
+    "name": "Holocene Hardfork - Proof Contract Upgrades",
+    "description": "Upgrades the `MIPS.sol`, `FaultDisputeGame.sol`, and `PermissionedDisputeGame.sol` contracts for Holocene."
```

Comment:
where is the MIPS upgrade? I can't see any storage referring to this contract?

Reply:
MIPS is implicitly upgraded by upgrading the games. The games reference the new MIPS implementation. This is also part of the validations. There's no MIPS proxy contract that needs upgrading, it's a singleton that is referenced directly from DG contracts.